build_migration_photos_list.py

```diff
@@ -1,45 +1,55 @@
 from __future__ import print_function
```

Comment:
create pickle file for each album (photoset) in flickr.  record which albums have been built to make the process restartable.